service/collector/chart_resource.go

```diff
@@ -80,6 +91,21 @@ func (c *ChartResource) Collect(ch chan<- prometheus.Metric) error {
 			key.ReleaseStatus(chartConfig),
 			key.Namespace(chartConfig),
 		)
+
+		if key.IsCordoned(chartConfig) {
+			t, err := convertToTime(key.CordonUntil(chartConfig))
+			if err != nil {
+				c.logger.Log("level", "warning", "message", "could not convert cordon-until", "stack", fmt.Sprintf("%#v", err))
+				continue
```

Comment:
Skipping in case of parsing error since we need to update other resource status.

Reply:
We could track error metrics here to reason about the processing. Just as an idea.